scripts/check-package-version.cjs

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * This script is used in CI to compare the version in a package's package.json to the expected version we're releasing,
+ * which is set in the SETVERSION_VERSION environment variable.
+ */
+
+const fs = require("fs");
+
+if(process.env.SETVERSION_VERSION === undefined) {
+  console.error("SETVERSION_VERSION env variable is undefined");
+  process.exit(1);
+}
+
+const json = JSON.parse(fs.readFileSync("./package.json"));
```

Comment:
might as well use readJsonSync here from fs-extra. The example for it is even checking package version like this code: https://github.com/jprichardson/node-fs-extra/blob/master/docs/readJson-sync.md#example